src/common/power.cpp

```diff
@@ -0,0 +1,115 @@
+// PoWER uses Equi-X by tevador here:
+// <https://github.com/tevador/equix/>.
+//
+// Equi-X is:
+// Copyright (c) 2020 tevador <tevador@gmail.com>
+//
+// and licensed under the terms of the LGPL version 3.0:
+// <https://www.gnu.org/licenses/lgpl-3.0.html>
```

Comment:
https://github.com/tevador/equix/blob/master/LICENSE

https://gitlab.torproject.org/tpo/core/arti/-/blob/main/doc/LGPL-and-rust.md